package.json

```diff
@@ -5,16 +5,22 @@
   "main": "index.js",
   "scripts": {
     "coverage": "npm run db-migrate && mocha --reporter mochawesome --timeout=20000 --require babel-register --exit",
-    "db-migrate": "sequelize db:migrate:undo:all && sequelize db:migrate",
+    "db-migrate": "node_modules/.bin/sequelize db:migrate:undo:all && sequelize db:migrate",
```

Comment:
this should not be referenced from node modules. The sequelize cli in dependencies should handle this.

Reply:
Also, in production, we don't want to drop and run all migrations whenever we deploy to production. What happens to our data then? There's a way to only run migrations only when there's a migration to be run. Find out how. @leksyib

Reply:
Thanks, @9jaswag  will do.